tcms/static/js/tcms_actions.js

```diff
@@ -119,8 +119,7 @@ var default_messages = {
     'last_case_run': 'It is the last case run',
     'bookmark_added': 'Bookmark added.',
     'no_run_selected': 'No run selected.',
-    'invalid_bug_id': 'Please input a valid bug id which is less equal 7 numberes, e.g. 1001234',
-    'invalid_jira_id': 'Please input a valid JIRA id, e.g. TCMS-32',
+    'invalid_bug_id': 'Please input a valid bug id!',
```

Comment:
Like what for example? At this moment we don't have any idea what the underlying bug tracker is. The only property Nitrate used previously to distinguish between Bugzilla and JIRA ids was the presence of dashes in the IDs which is totally wrong. In fact in Bugzilla you can give aliases to bugs and you can have descriptive bug names including dashes.  You see that this logic breaks even without my patch.